.github/pull_request_template.md

```diff
@@ -0,0 +1,57 @@
+## Overview
+<!--  
+Provide a high-level description of this change.   
+-->
+
+
+
+## Type of change
+<!--  
+Check the box below that describes your change best:
+--> 
+
+- [ ] Created a new plugin
+- [ ] Improved an existing plugin
+- [ ] Fixed a bug in an existing plugin
+- [ ] Improved contributor utilities or experience
+
+## Related Issue(s)
+<!--  
+If applicable - add the issue that your PR relates to or closes:
+  - use Resolves: #ISSUE_NUMBER to trigger closing of the issue on merge of this PR  
+  - use Relates: #ISSUE_NUMBER to indicate relation to an issue, but issue will not close  
+-->  
+
+* Resolves: #
+* Relates: #
+
+## How To Test
+<!--
+Provide testing instructions for validating the changes introduced in this PR.
+This will serve as a starting point for your reviewers, for functional testing.
+-->
```

Comment:
```suggestion
<!--
Provide testing instructions for validating the changes introduced in this PR.
This will serve as a starting point for your reviewers, for functional testing.

If you created a new plugin, you can add a command here which can be used to test authentication.
For example, for the AWS CLI:
  aws s3 ls
-->
```